packages/ui/src/common/components/page/Sidebar/Breadcrumbs/BreadcrumbsList.tsx

```diff
@@ -1,17 +1,27 @@
 import React from 'react'
+import { useHistory } from 'react-router'
 import styled from 'styled-components'
+import useBreadcrumbs from 'use-react-router-breadcrumbs'
 
-import { BreadcrumbsItem, BreadcrumbsItemProps } from './BreadcrumbsItem'
+import { knownRoutes, excludePaths } from '../../../../model/breadcrumbs'
+
+import { BreadcrumbsItem } from './BreadcrumbsItem'
 
 export interface BreadcrumbsListProps {
-  crumbs: Array<BreadcrumbsItemProps>
+  objectName?: string
 }
 
-export function BreadcrumbsList({ crumbs }: BreadcrumbsListProps) {
+export function BreadcrumbsList({ objectName }: BreadcrumbsListProps) {
+  const { location } = useHistory()
+  const routesList = objectName ? [...knownRoutes, { path: location.pathname, breadcrumb: objectName }] : knownRoutes
+  const crumbs = useBreadcrumbs(routesList, { excludePaths })
+  console.log(crumbs)
   return (
     <BreadcrumbsListComponent>
-      {crumbs.map(({ href, text }, index) => (
-        <BreadcrumbsItem key={index} href={href} text={text} />
+      {crumbs.map(({ match: { url }, breadcrumb, key }) => (
+        <BreadcrumbsItem key={key} url={url}>
+          {breadcrumb}
+        </BreadcrumbsItem>
```

Comment:
Could you also make the last breadcrumb text only? Right now it is disabled only but still renders as a link on the page.